src/Mvc/test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/Script.cshtml

```diff
@@ -149,12 +149,12 @@
     </script>
 
     <!-- Globbed script tag with existing files and version -->
-    <script asp-src-include="**/*.js" asp-append-version="true">
+    <script asp-src-include="**/styles/**/*.js" asp-append-version="true">
```

Comment:
Do we actually put JavaScript under the `styles` directory? Seems weird...

Reply:
I don't really have any more context than you, but this is the expected styles directory https://github.com/dotnet/aspnetcore/tree/main/src/Mvc/test/WebSites/HtmlGenerationWebSite/wwwroot/styles, the issues it that all the test websites get their wwwroots combined when I did the helix conversion, since they all got squashed together, so **/*.js finds a bunch of extra files.  I don't really have a simple fix for that, so I just updated it to look at the styles directory :shrug: